sn_networking/src/cmd.rs

```diff
@@ -858,6 +865,17 @@ impl SwarmDriver {
     }
 
     fn try_interval_replication(&mut self) -> Result<()> {
+        let time_since_last_replication = self.last_replication_time.elapsed();
+        if time_since_last_replication < REPLICATION_THROTTLE_INTERVAL {
```

Comment:
the `per target` timeout is 45s, meanwhile this `throttle interval` is 60s,
which means `per target` becomes useless, as totally `covered` by this `throttle interval`.

I think this `over all throttle` is an overshot, and we only need `per target` level timeout,
which allows a new node to get updated asap, meanwhile retain the `low frequency of replication`